server.js

```diff
@@ -0,0 +1,117 @@
+const express = require('express');
+const db = require('./data/db');
+const server = express();
+server.use(express.json());
+const port = 5000;
+
+server.get('/', (req, res) => {
+  // 1st arg: route where a resource can be interacted with
+  // 2nd arg: callback to deal with sending responses
+  res.send('Hello from express');
+});
+
+server.post('/api/users', (req, res) => {
+  const {
+    name,
+    bio
+  } = req.body;
+  db
+    .insert({
+      name,
+      bio
+    })
+    .then(response => {
+      res.json(response);
+    })
+    .catch(error => {
+      res.json(error);
+    });
+});
+
+server.get('/api/users', (req, res) => {
+  db
+    .find()
+    .then(users => {
+      res.json({
+        users
+      });
+    })
+    .catch(error => {
+      res.json({
+        error
+      });
+    });
+});
+
+server.get('/api/users/:id', (req, res) => {
+  // pull id off of req.params;
+  // invoke proper db.method(id) passing it the id.
+  // handle the promise like above
+  const {
+    id
+  } = req.params;
+
```

Comment:
Don't forget to set up an error message for if there's no user with that ID. The error in your catch is a server error, it doesn't cover the user not existing.